lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -69,6 +69,9 @@ define((require, exports, module) ->
 
     constructor: (@editor, @jqEditorContainer, @popoverHandler, @equationRangeHandler, @getFormulaElement) ->
       @contextPreviewExists = false
+      @rangeExists = false
+      @rangeCorrect = false
+      @currentRange = null
```

Comment:
It is exactly like that, yeah. I just thought it would be dirty to check if something is null all the time. But ok, I'll refactor it.